tqecjs/src/workspace/plaquettes/PlaquetteClass.js

```diff
@@ -297,25 +301,27 @@ export default class Plaquette extends Graphics {
 					newQubits.push(q);
 				}
 			}
-
-			// Create a new plaquette
-			let color = PlaquetteColors.Yellow;
-			if (this.color !== PlaquetteColors.Purple) {
-				color = PlaquetteColors.Purple;
-			}
-			console.log(color);
-			const newPlaquette = new Plaquette(
-				newQubits,
-				this.workspace,
-				this.gridSize,
-				color
-			);
-
-			// Add the plaquette to the tile
-			this.addChild(newPlaquette);
+			this.createNewPlaquette(newQubits);
 		}
+		
 	};
 
+	createNewPlaquette(newQubits) {
+		let newColor = PlaquetteColors.Yellow;
+		if (this.color !== PlaquetteColors.Purple) {
+			newColor = PlaquetteColors.Purple;
+		}
+		console.log("New plaquette color = " + newColor.description);
+		const newPlaquette = new Plaquette(
+			newQubits,
+			this.workspace,
+			this.gridSize,
+			newColor
+		);
+		// Add the plaquette to parent container
+		this.parent.addChild(newPlaquette);
```

Comment:
Adding to the parent for the first plaquette would add it to the tileClass, but consequent plaquettes would still be in the child plaquette. This causes issues when clearing as mentioned in #29